script.rb

```diff
@@ -3,19 +3,11 @@
 require './twitter_account'
 
 spreadsheet_url = ENV['SPREADSHEET_URL']
-twitter_users = [ENV['TWITTER_ID1'],ENV['TWITTER_ID2']]
 
-twitter_users.each do |twitter_user|
-  if twitter_user == ENV['TWITTER_ID1']
-    twitter_password = ENV['TWITTER_PASS1']
-    worksheet_name = ENV['WORKSHEET_NAME']
-  elsif twitter_user == ENV['TWITTER_ID2']
-    twitter_password = ENV['TWITTER_PASS2']
-    worksheet_name = ENV['WORKSHEET_NAME2']
-  end
-  analytics_client = TwitterAnalyticsClient.new(twitter_user, twitter_password)
+TwitterAccount.all.each do |twitter_user|
+  analytics_client = TwitterAnalyticsClient.new(twitter_user)
```

Comment:
I change class name `TwitterUser` later.(Because I should change database name, also..)